{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py

```diff
@@ -184,10 +172,12 @@
 CELERY_RESULT_SERIALIZER = 'json'
 {% endif %}
 
+## Sentry
+
 if env('SENTRY_DSN', default=''):
     sentry_logging = LoggingIntegration(
         level=logging.INFO,  # Capture info and above as breadcrumbs
-        event_level=logging.ERROR     # Send error events from log messages
+        event_level=logging.ERROR  # Send error events from log messages
     )
 
     sentry_sdk.init(
```

Comment:
Can we add a default logging configuration?

Reply:
What do you mean?